api/app/Models/PoolCandidate.php

```diff
@@ -194,17 +195,6 @@ private function filterByClassificationToSalary(Builder $query, array $classific
         return $query->whereRaw('EXISTS (' . $sql . ')', $parameters);
     }
 
-    public function filterByCmoAssets(Builder $query, array $cmoAssets): Builder
```

Comment:
Uh, oh, I think you broke "old search".  Feature flag migration strikes again!
![image](https://user-images.githubusercontent.com/8978655/196534493-749ea5a4-dd09-4b3e-a10b-208e35c71561.png)